pkgs/games/keeperrl/default.nix

```diff
@@ -0,0 +1,57 @@
+{ stdenv, fetchFromGitHub, SDL2, SDL2_image, openal, libvorbis, zlib, curl,
+  libGL, makeWrapper, boost, makeDesktopItem }:
+
+stdenv.mkDerivation rec {
+  pname = "keeperrl";
+  version = "2019-10-21";
+
+  src = fetchFromGitHub {
+    owner = "miki151";
+    repo = "keeperrl";
+    rev = "74b89ea";
+    sha256 = "17rplprichf0l52lbiampp78g8rz3msclkyja3v8kb79nxy93fns";
+  };
+
+  patches = [ ./makefile_cflags.patch ];
+
+  buildInputs = [
+    SDL2 SDL2_image openal libvorbis zlib curl libGL makeWrapper
+  ] ++ stdenv.lib.optional stdenv.isDarwin boost ;
+
+  # Needed so the builder finds SDL.h
+  CFLAGS = "-I${SDL2.dev}/include/SDL2";
```

Comment:
I think `NIX_CFLAGS_COMPILE = [ "-I ..." ]` should be used instead, which will end up in CFLAGS but with some extra stuff.